src/Simplex/Messaging/Server/CLI.hs

```diff
@@ -33,12 +35,10 @@ exitError :: String -> IO ()
 exitError msg = putStrLn msg >> exitFailure
 
 confirmOrExit :: String -> IO ()
-confirmOrExit s = do
-  putStrLn s
-  putStr "Continue (Y/n): "
-  hFlush stdout
-  ok <- getLine
-  when (ok /= "Y") exitFailure
+confirmOrExit s =
+  withPrompt (s <> "\nContinue (Y/n): ") $ do
+    ok <- getLine
+    when (ok /= "Y") exitFailure
```

Comment:
It's better to print something before `exitFailure`, it's not entirely clear files aren't deleted if you type "y" lowercase.. (it just does nothing). Or at least compare with it as well.
"Aborting deletion" can be passed to `msg` here.
```suggestion
confirmOrExit :: String -> String -> IO ()
confirmOrExit s msg =
  withPrompt (s <> "\nContinue (Y/n): ") $ do
    ok <- getLine
    when (ok /= "Y" && ok /= "y") $ exitError msg
```

Reply:
It's not changed really, and we didn't accept lowercase Y before - just in case

Reply:
added output that nothing was deleted